modules/metering-reports.adoc

```diff
@@ -0,0 +1,332 @@
+cluster-memory-utilization                   23m
+cluster-persistentvolumeclaim-request        23m
+namespace-cpu-request                        23m
+namespace-cpu-usage                          23m
+namespace-cpu-utilization                    23m
+namespace-memory-request                     23m
+namespace-memory-usage                       23m
+namespace-memory-utilization                 23m
+namespace-persistentvolumeclaim-request      23m
+namespace-persistentvolumeclaim-usage        23m
+node-cpu-allocatable                         23m
+node-cpu-allocatable-raw                     23m
+node-cpu-capacity                            23m
+node-cpu-capacity-raw                        23m
+node-cpu-utilization                         23m
+node-memory-allocatable                      23m
+node-memory-allocatable-raw                  23m
+node-memory-capacity                         23m
+node-memory-capacity-raw                     23m
+node-memory-utilization                      23m
+persistentvolumeclaim-capacity               23m
+persistentvolumeclaim-capacity-raw           23m
+persistentvolumeclaim-phase-raw              23m
+persistentvolumeclaim-request                23m
+persistentvolumeclaim-request-raw            23m
+persistentvolumeclaim-usage                  23m
+persistentvolumeclaim-usage-raw              23m
+persistentvolumeclaim-usage-with-phase-raw   23m
+pod-cpu-request                              23m
+pod-cpu-request-raw                          23m
+pod-cpu-usage                                23m
+pod-cpu-usage-raw                            23m
+pod-memory-request                           23m
+pod-memory-request-raw                       23m
+pod-memory-usage                             23m
+pod-memory-usage-raw                         23m
+----
+
+ReportQueries with the `-raw` suffix are used by other ReportQueries to build more complex queries, and should not be used directly for reports.
+
+`namespace-` prefixed queries aggregate Pod CPU/memory requests by namespace, providing a list of namespaces and their overall usage based on resource requests.
+
+`pod-` prefixed queries are similar to 'namespace-' prefixed, but aggregate information by Pod, rather than namespace. These queries include the Pod's namespace and node.
+
+`node-` prefixed queries return information about each node's total available resources.
+
+`aws-` prefixed queries are specific to AWS. Queries suffixed with `-aws` return the same data as queries of the same name without the suffix, and correlate usage with the EC2 billing data.
+
+The `aws-ec2-billing-data` report is used by other queries, and should not be used as a standalone report. The `aws-ec2-cluster-cost` report provides a total cost based on the nodes included in the cluster, and the sum of their costs for the time period being reported on.
+
+For a complete list of fields each report query produces, use `kubectl` to get the object as YAML, and check the `columns` field:
```

Comment:
fields, each

Reply:
I'm fairly confident we don't want a comma after `fields` unless we change the sentence. We could maybe use:
> For a complete list of fields, use `kubectl` to get the ReportQuery as YAML, and check the `spec.columns` field:

Reply:
Sorry about that! You're correct. I misread the usage.